src/app/components/sandbox/CodeEditor/Monaco.js

```diff
@@ -46,14 +48,6 @@ const Container = styled.div`
   z-index: 30;
 `;
 
-const fontFamilies = (...families) =>
```

Comment:
Let's comment this part of the code out instead, I'm hoping to enable it next week again (it's not used because of a bug).

Reply:
Restored and commented.